pkg/storage/db.go

```diff
@@ -120,6 +140,41 @@ func (db *DB) Init(ctx context.Context, postgresConfig *PostgresConfig) error {
 	return initerr
 }
 
+func (db *DB) doesDBExist(pgdb *sql.DB, dbName string) bool {
+	row := pgdb.QueryRow("SELECT datname FROM pg_catalog.pg_database WHERE lower(datname) = lower('" + dbName + "');") // nolint
+	if err := row.Scan(); err != nil {
+		return false
+	}
+
+	return true
+}
+
+func (db *DB) create(pgdb *sql.DB, name string) error {
```

Comment:
nit: it doesn't look like this needs to be a function of `db` since it's not dealing with any of that struct's internal state

Reply:
I've changed the logic a bit so that it _is_ part of the internal `db` struct